src/components/FilterOptionsList/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./index.css";
+
+// This renders the filter options of a given filter as parameter.
+// It is called in AdvancedFilters component
+const FilterOptionsList = ({ filter }) => {
+  return (
+    <div className="filterOptionsList">
+      {filter.filterParameters.map((option) => {
+        return (
+          <div
+            className="filterOption d-flex align-items-center p-1"
```

Comment:
We don't prefer inline styling.

Reply:
d-flex align-items-center p-1 => they are bootsrap properties.

Reply:
@fadime-ozdemir this is not inline styling its classNames for bootstrap so its okai you dont need to change